src/terminal/init.lua

```diff
@@ -18,14 +18,38 @@ local M = {
   _DESCRIPTION = "Cross platform terminal library for Lua (Windows/Unix/Mac)",
 }
 
+--- Returns the height and width of the terminal screen.
+-- @treturn number, number the height in rows and the width in columns
+function M.get_dimensions()
+  local size = sys.get_terminal_size()
+  return size.height, size.width
+end
+
+--- Sets the cursor position and writes it to the terminal, without pushing onto the stack.
+-- @tparam number row
+-- @tparam number column
+-- @return true
+-- @within cursor_position
+function M.cursor_set(row, column)
```

Comment:
this function is overwritten by the implementation below. So this can go, correct? and then the `get_dimensions` above is also no longer used?